guiEditor/src/diagram/workbench.tsx

```diff
@@ -1015,6 +1034,34 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         this.props.globalState.onGizmoUpdateRequireObservable.notifyObservers();
     }
 
+    // Move the selected controls. Can be either on horizontal (leftInPixels) or 
+    // vertical (topInPixels) direction
+    moveControls(horizontal: boolean, amount: number) {

```

Comment:
should second parameter be named vertical?

Reply:
seems like horizontal is the mode (i.e. horizontal when true, vertical when false?).

Reply:
Yeah, you're right, it's the mode! I think `moveHorizontal` could work, what do you think?

Reply:
Updated!

Reply:
Ah got it, I misread it, thanks for the explanation!